### ISSUE TITLE
Is it possible to edit an opened excel file and see the updates on the opened excel file?

### ISSUE THREAD

**Issue by ArturasDruteika** [troldal/OpenXLSX#256]
Before describing my issue, I just want to tell that this lib is really great and helps us with our solutions which require reports saving.

My question is wheather or not is it possible to open a created excel file (which on the program is not closed) and see a live update when some value is being written to the opened excel file?

When I open the excel file (not from the program but using something like libre office or any other program) I do not see the live updates to the excel file when my application writes a new report (when the application populates some cells with some new data). While I keep the file open I don not see any updates, but once I close it and if the application inputs new data then once I open the excel file again I can see the newly written data on the excel file.

**Comment by aral-matrix** [comment]
No, unfortunately that is not possible without a lot of unnecessary loading operations:
The way XLSX files are stored is in a zip archive of XML files. When an update to that file is made in an application, that application has to *save* the file to disk for the changes to become available to any other application.

What you can do, is configure an inotify watch (on Linux - this doesn't work well on Windows) on the file/directory that you want to monitor, and close & re-open it in OpenXLSX when that watch detects a change to the file.

However, I don't think this is a very practical solution. And interacting with Excel from "outside" of MS Office is non-trivial (but probably involves visual basic for applications) - for any other application, that totally depends on that app.

If you don't mind, I would close this issue in a few days.

**Comment by ArturasDruteika** [comment]
Thank you for the information.

**Comment by aral-matrix** [comment]
Sorry to have no better news - but that functionality is beyond the powers of a library that works on file basis.

However, stay tuned for an update related to supporting cell formatting and merging :)
